src/searcher.rs

```diff
@@ -29,20 +32,30 @@ impl Searcher {
     /// search results.
     ///
     /// Raises a ValueError if there was an error with the search.
+    #[args(limit = 10, sort_by = "SORT_BY")]
     fn search(
```

Comment:
We cannot have the matching document count with this setting though :-S

Reply:
By matching count you mean using the `CountCollector`? If so, yeah I'm aware, adding that would probably mean switching to the `Multicollector`. I will probably do so at the next iteration of this.